spec/models/game_spec.rb

```diff
@@ -74,4 +74,61 @@
       expect(game.active_player).to eq(game.black_player)
     end
   end
+
+  describe '#stalemate?' do
+    before do
+      allow_any_instance_of(Game).to receive(:populate_board!).and_return true
+    end
+
+    let(:game) { FactoryGirl.create(:game) }
+    let(:white_king) { FactoryGirl.create(:king, type: 'King', color: 'white', row: 7, column: 3) }
+    let(:black_king) { FactoryGirl.create(:king, type: 'King', color: 'black', row: 0, column: 3) }
+
+    it 'should return true if active player is in stalemate' do
+      game.pieces << white_king
+      game.pieces << black_king
```

Comment:
i did add `game: game` to both `white_king` and `black_king`, and eliminate line 88 and 89  only fail the test so i leave it as it was originally.